src/Utils/TypeInfo.php

```diff
@@ -157,6 +157,9 @@ public static function extractTypes($type, ?array $typeMap = null)
         if ($type instanceof WrappingType) {
             return self::extractTypes($type->getWrappedType(true), $typeMap);
         }
+
+        $type = Type::resolveLazyType($type);
```

Comment:
I would enforce type to be already resolved in here. So this should be removed. If we correctly resolve types in `UnionType::getTypes`, `ObjectType::getInterfaces`, `FieldDefinition::getType`, etc then this method will always deal with fully loaded types (and it should)

Reply:
Right!